projects/ngx-d3timeline/src/lib/options.service.ts

```diff
@@ -6,11 +6,13 @@ import { TimelineView } from './view/timeline-view';
 export class OptionsService {
   getTranslation(
     range: number,
+    cx: number,
```

Comment:
I think this function is doing too much, better to put `cx` and `cy` as `dx` and `dy` into the `Tick` class

Reply:
See above reply to similar comment

Reply:
Would rather just remove the code and have it drawn incorrectly than keep the code in.